text/0027-manual-disabling-of-flaky-tests.md

```diff
@@ -0,0 +1,73 @@
+* Start Date: 2022-10-13
+* RFC Type: informational
+* RFC PR: https://github.com/getsentry/rfcs/pull/27
+
+# Summary
+
+flaky tests in `sentry` / `getsentry` significantly impact developer
+productivity.  ideally an automated system would improve this situation but
+in the interim this rfc aims to document a manual process for disabling
+flaky tests.
+
+# Motivation
+
+currently any test flake will require the entire job to be rerun, which
+usually wastes about ~20 minutes of developer time per test flake.
+
+to minimize disruptiveness, this rfc proposes manually disabling with a
+procedure.
+
+# Supporting Data
+
+https://sentry.io/organizations/sentry/dashboard/7997/?project=5350637&project=4857230&project=2423079&statsPeriod=14d
+
+# Options Considered
+
+## manual disabling process
+
+- identify the owner of the test in question
+    - often `.github/CODEOWNERS` can help
+    - otherwise utilize `git log -- path/to/the/test/file`
+- find the corresponding sentry issue:
+    - for example: [django tests](https://sentry.io/organizations/sentry/issues/?limit=5&project=2423079&query=&sort=freq&statsPeriod=14d)
+- create a JIRA ticket (or if the team uses github issues, do that instead)
```

Comment:
Should we be consistent here to reduce the overhead in this process to make it easier for the party disabling the test?
ie. Create a jira project for all flakey tests, or always use github issues with a specific label?

Also this way the skip reasons are easy to connect back to their corresponding issue (instead of having to hunt both Jira & Github)

Reply:
without assigning them directly to teams there won't be accountability and they won't be in team's sprint boards -- from experience if disabled tests aren't owned they won't get fixed

Reply:
Fair enough 👍, should we push teams to publish where to put the issue at least?

Reply:
yeah I think "ownership" is a bit of an unsolved problem here at sentry -- ideally there would be a consistent and reliable way to create an issue for a team but I think that's a bit orthogonal to this (though it does make this step the most difficult of this manual process)